user/views.py

```diff
@@ -4,15 +4,17 @@
 from rest_framework.authtoken.views import ObtainAuthToken
 from rest_framework.settings import api_settings
 
```

Comment:
There should be a blank line between import statements and the rest of the code according to the code style guidelines.